### DISCUSSION TITLE
Notes do not update and Trilium hangs

### DISCUSSION THREAD

**Discussion by nielsnw** [TriliumNext/Trilium#2925] [General]
Hello,

I dont know if this has been addressed already, could not find anything related to this issue.

When I add text to a note Trilium seems to hang on that note: when selecting a different one, the note's name changes but the contents of the old note are displayed. When I refresh my browser the text I entered is gone and correcte contents are displayed again when I change notes. Also I can enter text but deleting it or hitting a Carriage return does not work anymore.

I feel this happens when I use a lot of bullet or numbered lists and/or add images to a note. Addition: when this happens it seems I can add new notes and content and this saves correctly. The issue seems to be with those select notes with lists and effects the entire application until I reload the frontend.

Looking forward to a solution, I use Trilium alot.

Kind regards, Niels

App version: 0.47.5
DB: 183
Sync: 20
Build revision: 2021-06-25T21:26:48+02:00

**Comment by zadam** [comment]
Hi, that looks like a CKEditor bug - sometimes it crashes and then it's sort of stuck like you describe.

You use quite old version, upgrading to the latest might help.
